src/components/Footer.jsx

```diff
@@ -57,9 +62,29 @@ class SiteFooter extends React.Component {
             <img
               style={{ maxHeight: 45 }}
               src={logo || config.LOGO_TRADEMARK_URL}
-              alt={intl.formatMessage(messages['footer.logo.altText'])}
+              alt={process.env.LOGO_ALT_TEXT || intl.formatMessage(messages['footer.logo.altText'])}
             />
           </a>
+          )}
+          <div className="copyright-col">
+            {process.env.TRADEMARK_TEXT
+            && (
+            <div className="text-gray-500 small">
+                {process.env.TRADEMARK_TEXT}
+            </div>
+            )}
+            <div>
+              <ul className="footer-sub-nav">
+                {this.renderLinkIfExists(process.env.ABOUT_US_URL, 'About Us')}
+                {this.renderLinkIfExists(process.env.TERMS_OF_SERVICE_URL, 'Terms of Service')}
+                {this.renderLinkIfExists(process.env.PRIVACY_POLICY_URL, 'Privacy Policy')}
+                {this.renderLinkIfExists(process.env.HONOR_CODE_URL, 'Honor Code')}
+                {this.renderLinkIfExists(process.env.CONTACT_URL, 'Contact Us')}
+                {this.renderLinkIfExists(process.env.ACCESSIBILITY_URL, 'Accessibility')}
+                {this.renderLinkIfExists(process.env.SUPPORT_CENTER_URL, process.env.SUPPORT_CENTER_TEXT || 'FAQ & Help')}
```

Comment:
We should use formatMessage with internationalization. An example of that can be seen in [dashboard MFE](https://github.com/openedx/frontend-app-learner-dashboard/blob/master/src/containers/LearnerDashboardHeader/BrandLogo.jsx#L11C11-L11C24)